src/pages/energiatodistus.svelte

```diff
@@ -38,12 +38,11 @@
     );
   };
 
-  const selectShownKatuosoite = (l, perustiedot) => {
-    if (l == 'sv' && perustiedot['katuosoite-sv'])
-      return perustiedot['katuosoite-sv'];
-    else if (l == 'fi' && !perustiedot['katuosoite-fi'])
-      return perustiedot['katuosoite-sv'];
-    else return perustiedot['katuosoite-fi'];
+  const selectByLocaleOrAvailable = (lang, fiLabel, svLabel) => {
+    if (lang === 'sv' && svLabel) return svLabel;
+    else if (lang === 'fi' && !fiLabel) return fiLabel;
+    else if (fiLabel) return fiLabel;
+    else return '';
```

Comment:
Olikos meillä tullut speksi tälle kaksikielisyydelle, jossa näytetään aina valitun localen mukaan?
Jos näin on, niin tämän funkkarin rajapinnasta saanee hieman vähemmän intrusiivisen jos se ottaisi sisään localen, labelin ja objektin.
`selectByLocaleOrAvailable($locale, 'katuosoite', energiatodistus.perustiedot)`
Eli tehdään tuo -fi/-sv säätö itsessään funktion sisällä, niin tuo funkkarikutsun manuaalinen työ pieneenee huomattavasti.